app/Http/Controllers/OrganizationUserController.php

```diff
@@ -2,84 +2,86 @@
 
 namespace App\Http\Controllers;
 
+use App\Http\Resources\OrganizationUserResource;
 use App\OrganizationUser;
+use Illuminate\Contracts\Foundation\Application;
+use Illuminate\Contracts\Routing\ResponseFactory;
+use Illuminate\Http\JsonResponse;
 use Illuminate\Http\Request;
+use App\Repository\OrganizationUsers\OrganizationUserInterface;
+use Illuminate\Http\Response;
+use Illuminate\Support\Facades\Auth;
 
 class OrganizationUserController extends Controller
 {
     /**
-     * Display a listing of the resource.
-     *
-     * @return \Illuminate\Http\Response
+     * @param Request $request
+     * @param OrganizationUserInterface $model
+     * @return Application|ResponseFactory|JsonResponse|Response
      */
-    public function index()
+    public function passwordReset(Request $request, OrganizationUserInterface $model)
     {
-        //
-    }
+        try{
+            $response = $model->resetUserPassword(
+                $request->newPassword
+            );
+            if($response == false){
```

Comment:
Falsy values are usually checked with a triple equal to sign (===)

Reply:
changed